website/static/js/addProjectPlugin.js

```diff
@@ -89,6 +96,15 @@ var AddProject = {
             var success = function _success (result) {
                 self.viewState('success');
                 self.goToProjectLink(result.data.links.html);
+                if (self.newProjectInstitutions.length > 0){
```

Comment:
I'm a bit confused why `newProjectInstitutions` is necessary; couldn't you just use `self.checkInstitutions`?